releasing/cloudbuild.yaml

```diff
@@ -2,53 +2,53 @@
 #   [\w/]+/v\d+\.\d+\.\d+
 #
 steps:
-- name: 'bash'
-  args:
-  - 'echo'
-  - 'Cloud build substitution check: '
-  - 'BUILD_ID=$BUILD_ID'
-  - 'PROJECT_ID=$PROJECT_ID'
-  - 'REVISION_ID=$REVISION_ID'
-  - 'REPO_NAME=$REPO_NAME'
-  - 'COMMIT_SHA=$COMMIT_SHA'
-  - 'BRANCH_NAME=$BRANCH_NAME'
-  - 'TAG_NAME=$TAG_NAME'
+  - name: "bash"
```

Comment:
Is there a reason you committed this file? This relates to the releasing process and I'd prefer it to not be modified unless it's necessary.

Reply:
I think my editor was lint to this yaml file.
I restore it from master.